### ISSUE TITLE
Unable to get plex token during setup wizard with baseurl set

### ISSUE THREAD

**Issue by niayh** [tidusjar/PlexRequests.Net#615]
#### Plex Requests.Net Version:
1.9.4

#### Operating System:
Ubuntu 16.04

#### Mono Version:
4.2.1 and 4.6.1

#### Applicable Logs (from `/logs/` directory or the Admin page):

```
2016/10/25 20:30:46.214 PlexRequests.Services.Jobs.SickRageCacher Trace: Getting the settings 
2016/10/25 20:30:46.221 PlexRequests.Services.Jobs.StoreBackup Trace: Starting DB Backup 
2016/10/25 20:30:46.216 PlexRequests.Services.Jobs.CouchPotatoCacher Trace: Getting the settings 
2016/10/25 20:30:46.233 PlexRequests.Services.Jobs.PlexAvailabilityChecker Warn: A setting is null, Ensure Plex is configured correctly, and we have a Plex Auth token. 
2016/10/25 20:30:46.526 PlexRequests.Services.Jobs.PlexAvailabilityChecker Debug: Validation of the plex settings failed. 
2016/10/25 20:30:46.238 PlexRequests.UI.Program Info: We are on Mono! 
2016/10/25 20:30:46.718 PlexRequests.Services.Jobs.StoreBackup Trace: Starting DB Cleanup 

```

#### Reproduction Steps:
Start Plex Requests from scratch/fresh install (remove PlexRequests.sqlite) and start the application with the command `mono PlexRequests.exe -b plexrequests`. Application starts fine and wizard shows up; however, we are unable to get the plex token after entering Plex login info. Error happens regardless of accessing PR.NET through the port (http://host:3579/plexrequests) or from the reverse proxy (http://host/plexrequests). Problem does not present without use of the -b flag and only happens once a baseurl is set.

**Comment by tidusjar** [comment]
OK thanks, must be an issue with finding the .js script

**Comment by whitecollar17** [comment]
Fixed in the release of 1.9.4
